apps/full-site-editing/full-site-editing-plugin/block-inserter-modifications/contextual-tips/tip-link.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * WordPress dependencies
+ */
+import { select } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { isGutenframed } from './utils';
+
+const isEditorIFramed = isGutenframed();
```

Comment:
How do you feel about moving that check in-line? If not, could we rename it to `inIframe` like https://github.com/Automattic/wp-calypso/blob/4537033c6794449ce0a10b69d72daf5fa658d553/apps/wpcom-block-editor/src/utils.js#L7-L19